frame/staking-rewards/src/benchmarking.rs

```diff
@@ -121,5 +133,19 @@ benchmarks! {
 
 	}: _(RawOrigin::Signed(user), position_id, validated_ratio)
 
+	reward_acumulation_hook_reward_update_calculation {
+		let reward = Reward {
+			asset_id: 1_u128.into(),
+			total_rewards: 0_u128.into(),
+			total_dilution_adjustment: 0.into(),
+			max_rewards: 100.into(),
+			reward_rate: RewardRate::per_second(1.into(), 1.try_into_validated().unwrap()),
+			last_updated_timestamp: 0,
+			claimed_rewards: 0_u128.into()
+		};
+	}: {
+		let reward = Pallet::<T>::reward_acumulation_hook_reward_update_calculation(1.into(), 1.into(), reward, T::UnixTime::now().as_secs());
```

Comment:
looks like, you added weights of `T::UnixTime::now()` (1 read from DB) in weights of `reward_acumulation_hook_reward_update_calculation`.

Reply:
Ah, I didn't realize that now() was a DB read. I'll extract it out, thanks!